DetectorDescription/DDCMS/interface/DDShapes.h

```diff
@@ -1,221 +0,0 @@
-#ifndef Detector_Description_DDCMS_DDShapes_h
```

Comment:
As I mentioned in #31578, this file could be emptied and replaced with:
```
using DDTrap = dd4hep::Trap;
using DDTubs = dd4hep::Tube;
using DDCons = dd4hep::ConeSegment;
using DDTruncTubs = dd4hep::TruncatedTube;
```
Then nothing else has to be changed. Or am I missing something?

Reply:
This is used by a MagneticField builder only. @namapane is working on an alternative solution anyway that is expected in a couple of days. There is no need to keep these files in DetetctorDescription system.

Reply:
well it will take a couple of days but as I said I have to squeeze these in my schedule so don;t extpect it be ready by Tuesday :-)

Reply:
Wednesday then? ;-)